### DISCUSSION TITLE
MAC Address Spoofing for WAN on FritzBox! 6660 Cable

### DISCUSSION THREAD

**Discussion by maexbower** [Freetz-NG/freetz-ng#607] [Allgemein]
Hi all,
I wondered if the cable modem in the FritzBox! 6660 Cable supports the changing of the MAC Address just like usually the Ethernet Interfaces support. Instead of the MAC which is shown on the Device, the FritzBox should use a custom MAC address for the Cable WAN connection.

Why do I want to do something like that? 
Let's say there are countries in this world where customer focus is even less than in Germany and you are forced to use the provider's undersized cable modem. With that, you have no chance to get any proof for bad service quality in their network as they can always blame your home network.

**Comment by fda77** [comment]
In the ar7.cfg you could change your external mac which is used to request (public) ips by provider: `grep override /var/flash/ar7.cfg` I used this with 6490 to get a new ipv6-prefix after changing the subnet length, the dhcp of my provider has a timeout of 2-3 weeks. Check after a reboot if the changes where saved correctly.
I'm not sure if this helps you. There should be another mac of the modem itself, which should not be changed as docsis uses a certificate with this mac
